ui/src/routes/devices.$id.settings.mouse.tsx

```diff
@@ -183,6 +231,44 @@ export default function SettingsKeyboardMouseRoute() {
           </div>
         </div>
       </div>
+      <div className="space-y-4">
+        <SettingsPageHeader
+          title="Keyboard"
+          description="Customize keyboard behaviour"
+        />
+        <div className="space-y-4">
+          <SettingsItem
+            title="Enable Keyboard Mapping"
+            description="Enables mapping of keys from your native layout to the layout of the target device"
+          >
+            <Checkbox
+              checked={kbMappingEnabled}
+              onChange={e => {
+                handleKeyboardMappingChange(e.target.checked);
+              }}
+            />
+          </SettingsItem>
+          <SettingsItem
+            title="Keyboard Layout"
+            description="Set keyboard layout (this should match the target machine)"
+          >
+            <SelectMenuBasic
+              size="SM_Wide"
+              label=""
+              // TODO figure out how to make this selector wider like the EDID one?, (done but not sure if in desired way.)
+              //fullWidth
+              value={keyboardLayout}
+              options={[
+                { value: "en-US", label: "US" },
+                { value: "en-GB", label: "UK" },
+                { value: "en-GB_apple", label: "UK (Apple)" },
+                { value: "de_DE", label: "German (T1)" },
```

Comment:
I think this should be `value: "de-DE"` to match the value in `ui/src/keyboardMappings/KeyboardLayouts.ts`